index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO MICROVERSE</title>
+    <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="wrap-section">
+        <header>
+            <nav>
+                <a href="#" class="my-logo">Aneal Laryea</a>
+                <i class="fa-solid fa-bars"></i>
+            </nav> 
+        </header>
+    
+        <section class="headline">
+            <div class="primary-text">
+                <h3>Hey There.
```

Comment:
Good job so far by adding some of the changes required by the previous reviewers. So, I noticed that the previous reviewers requested to use the h1 instead of h3 because the h1 is the star of our design - it captures the most attention so it should be emphasized. It can also be useful in the case of search engine optimization.